content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -268,6 +268,14 @@ Used on: Endpoints
 
 In Kubernetes clusters v1.21 (or later), the Endpoints controller adds this annotation to an Endpoints resource if it has more than 1000 endpoints. The annotation indicates that the Endpoints resource is over capacity.
 
+## scheduler.alpha.kubernetes.io/preferAvoidPods (deprecated)
```

Comment:
I prefer to not make “deprecated” part of the fragment identifier.
```suggestion
## scheduler.alpha.kubernetes.io/preferAvoidPods (deprecated) {#scheduleralphakubernetesio-preferavoidpods}
```

Reply:
Done